kubernetes/assets/dashboards/kubernetes_control_plane.json

```
@@ -0,0 +1,3044 @@
+{
+    "title": "kubernetes Control-Plane",
```

nit: title (capitalization at the start, no hyphen in control plane, suffixed with overview to match the other OOTB dashboards)

Might want to remove the hyphen from control plane in the description field on line 3 as well
```suggestion
    "title": "Kubernetes Control Plane Overview",
```